client/src/components/presentation/OAuthForm/OAuthForm.js

```diff
@@ -25,6 +27,14 @@ const OAuthForm = props => {
         <ButtonText>Sign in with GitHub</ButtonText>
       </GitHubButton>
 
+      <FacebookLogin
+          appId="566781370495932"
+          autoLoad={true}
+          fields="name,email,picture"
+          onClick={props.handleSubmi}
```

Comment:
Typo:
`onClick={props.handleSubmi}`

should be
`onClick={props.handleSubmit}`